### ISSUE TITLE
oraclejdk7 is not supported anymore in Travis

### ISSUE THREAD

**Issue by rubensa** [mojohaus/webstart#60]
The Oracle Jdk7 is broken on container based trusty.

See [Trusty with oraclejdk7, ends with: ERROR: JAVA_HOME is set to an invalid directory: /usr/lib/jvm/java-7-oracle #7884](https://github.com/travis-ci/travis-ci/issues/7884#issuecomment-308451879)

While JAVA_HOME points to Jdk7, the actual executable java is set to Jdk8:

- java -version: 1.8.0_151
- JAVA_HOME: /usr/lib/jvm/java-7-oracle

Oracle Jdk8 and OpenJdk7 builds are still fine.

**Build log (exceprt):**

```bash
[…]
$ jdk_switcher use oraclejdk7
Switching to Oracle JDK7 (java-7-oracle), JAVA_HOME will be set to /usr/lib/jvm/java-7-oracle
update-java-alternatives: directory does not exist: /usr/lib/jvm/java-7-oracle
[…]
$ mvn clean verify -B -e -V
The JAVA_HOME environment variable is not defined correctly
This environment variable is needed to run this program
NB: JAVA_HOME should point to a JDK not a JRE
[…]
```

**Travis.yml**

```yml
language: java
sudo: false
install: true

jdk: 
  - openjdk7
  - oraclejdk7
  - oraclejdk8

script:
  - mvn clean verify -B -e -V

cache:
  directories:
    - '$HOME/.m2/repository'

branches:
    except:
        - gh-pages

notifications:
email: true
```

**Workaround**

Remove Oracle Jdk7 builds.

**Comment by tchemit** [comment]
can you give me a MR to apply, thanks.--Enjoy and ShareTony Chemit
          
            
            On févr. 19 2018, at 2:58 pm, rubensa <notifications@github.com> wrote:
            
            
              The Oracle Jdk7 is broken on container based trusty.
See Trusty with oraclejdk7, ends with: ERROR: JAVA_HOME is set to an invalid directory: /usr/lib/jvm/java-7-oracle #7884
While JAVA_HOME points to Jdk7, the actual executable java is set to Jdk8:

java -version: 1.8.0_151
JAVA_HOME: /usr/lib/jvm/java-7-oracle

Oracle Jdk8 and OpenJdk7 builds are still fine.
Build log (exceprt):
[…]
$ jdk_switcher use oraclejdk7
Switching to Oracle JDK7 (java-7-oracle), JAVA_HOME will be set to /usr/lib/jvm/java-7-oracle
update-java-alternatives: directory does not exist: /usr/lib/jvm/java-7-oracle
[…]
$ mvn clean verify -B -e -V
The JAVA_HOME environment variable is not defined correctly
This environment variable is needed to run this program
NB: JAVA_HOME should point to a JDK not a JRE
[…]
Travis.yml
language: java
sudo: false
install: true

jdk: 
  - openjdk7
  - oraclejdk7
  - oraclejdk8

script:
  - mvn clean verify -B -e -V

cache:
  directories:
    - '$HOME/.m2/repository'

branches:
    except:
        - gh-pages

notifications:
email: true
Workaround
Remove Oracle Jdk7 builds.

**Comment by rubensa** [comment]
Sure, but current master does not compile cause merge https://github.com/mojohaus/webstart/commit/1cb91f11ccd0490b2ad2f5152b025b169ce237c6

Please, fix and I'll make a MR for **travis.yml**

**Comment by tchemit** [comment]
what the heck :( I did some merges, too much ! I will fix this--Enjoy and ShareTony Chemit
          
            
            On févr. 19 2018, at 3:23 pm, rubensa <notifications@github.com> wrote:
            
            
              Sure, but current master does not compile cause merge 1cb91f1
Please, fix and I'll make a MR for travis.yml

**Comment by rubensa** [comment]
I've seen you just fixed master and **travis.yml** so closing this issue.

**Comment by tchemit** [comment]
Yes I was on it :) --Enjoy and ShareTony Chemit
          
            
            On févr. 19 2018, at 4:15 pm, rubensa <notifications@github.com> wrote:
            
            
              Closed #60.
